web/profiles/joinup/js/site_wide_featured.js

```diff
@@ -24,7 +34,16 @@
         return;
       }
 
-      $(context).find('[data-drupal-featured]').once('featured').addClass('is-featured');
+      $(context).find('[data-drupal-featured]').once('featured').each(function () {
+        var $this = $(this);
+        $this.addClass('is-featured');
+        $this.addClass('listing__card--corner');
```

Comment:
Those 2 can be either changed or added in 1 go to save 1 DOM manipulation :)

Reply:
Done.